pystiche_papers/gatys_et_al_2017/_data.py

```diff
@@ -103,15 +102,15 @@ def images() -> data.DownloadableImageCollection:
                 "https://github.com/leongatys/NeuralImageSynthesis/blob/cced0b978fe603569033b2c7f04460839e4d82c4/Images/ControlPaper/fig3_content.jpg?raw=true",
                 file="schultenhof.jpg",
                 license=license(
-                    "https://upload.wikimedia.org/wikipedia/commons/8/82/Schultenhof_Mettingen_Bauerngarten_8.jpg"
+                    "https://upload.wikimedia.org/wikipedia/commons/8/82/Schultenhof_Mettingen_Bauerngarten_8.jpg",
```

Comment:
Same as above. If this was done in response to me saying "exploding things makes them easier to read", we are slightly on the wrong path. The statement is true if you have 2+ input arguments. Since we only have one here, there is no need to "explode".